concepts/05 UI Components/VectorMap/20 Providing Data/10 Data for Areas/30 Using a JSON object.md

```diff
@@ -1,4 +1,4 @@
-If you have your data as a JSON object, assign its <a href="http://en.wikipedia.org/wiki/Uniform_resource_identifier" target="_blank">URI</a> to the **layers**.**dataSource** property. This way, no other actions will be required.
+If you have your data as a JSON object, assign its <a href="http://en.wikipedia.org/wiki/Uniform_resource_identifier" target="_blank">URI</a> to the [layers](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/).[dataSource](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#dataSource) property.
```

Comment:
What do you mean by "If you have your data as a JSON object"? This phrase sounds like the data is a collection of JSON objects. Is this correct?

Reply:
Yes, I meant that. I will change it.